### PR TITLE
unix: sysctl size argument using ARRAY_SIZE macro.

### PR DESCRIPTION
**Pull request by devnexen**
Following up previous PR on NetBSD thus left one on purpose.

### CODE REVIEW

**Comment by devnexen** [comment]
Just proposal, but make things more related to the first argument.

File: src/unix/openbsd.c [hunk 1]
```diff
@@ -190,12 +190,12 @@ int uv_cpu_info(uv_cpu_info_t** cpu_infos, int* count) {
   uv_cpu_info_t* cpu_info;
 
   size = sizeof(model);
-  if (sysctl(which, 2, &model, &size, NULL, 0))
+  if (sysctl(which, ARRAY_SIZE(which), &model, &size, NULL, 0))
```
Comment (saghul) [inline src/unix/openbsd.c]: Not sure this is ok. Currently we pass 2 elements sometimes and other times we pass 3.

Comment (devnexen) [reply src/unix/openbsd.c]: I did not notice but normally the 0 ending value is not necessary.

Comment (bnoordhuis) [reply src/unix/openbsd.c]: The third element with KERN_CPTIME2 is the CPU number, isn't it?

Comment (devnexen) [reply src/unix/openbsd.c]: Yes I expected some hits and misses in this change :-)

**Comment by bnoordhuis** [comment]
CI run with PR rebased against v1.x: https://ci.nodejs.org/job/libuv-test-commit/1662/

**Comment by bnoordhuis** [comment]
Thanks David, landed in 2ab3dc15.

**Pull request closed without merging**